### PR TITLE
navigation-bar

### PR DESCRIPTION
**Pull request by sirtobiwan**

### CODE REVIEW

File: components/Layout/index.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import styled from 'styled-components';
+import Navigation from '../NavigationBar';
 
 export default function Layout() {
   return (
+    <>
     <StyledHeader>
       <h1>My Solar Calculator</h1>
     </StyledHeader>
+    </>
```
Comment (NikRoe) [inline components/Layout/index.js:10]: You don't need fragments here:

```suggestion
    <StyledHeader>
      <h1>My Solar Calculator</h1>
    </StyledHeader>
```

File: components/NavigationBar/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import Link from 'next/link';
+
+const NavigationBar = styled.nav`
+  display: flex;
+  justify-content: center;
+  background-color: #f1f1f1;
+  padding: 10px;
+`;
+
+const NavItem = styled.div`
```
Comment (NikRoe) [inline components/NavigationBar/index.js:12]: a paragraph would be more semantic here:

```suggestion
const NavItem = styled.p`
```

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi Tobias,

your app looks really good already! I left only two comments for you to take a look at.

Really great that you reuse some of your styled components.

In case of your Layout component, you could have a reusable component that you don't need to call in every page manually, if you follow this [example from Next.js](https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts).

I recommend taking a look at the link and try to implement the Layout component. This will make your code even cleaner!

Keep it up 🙌

**Comment by sirtobiwan** [comment]
Hi Niklas,

thanks for the feedback.

In this case I need to call the layout component on every page.

Everything else was updated

**Comment by NikRoe** [comment]
You would only need to call it in your `_app.js` and not on every page.

**Comment by sirtobiwan** [comment]
Exactly, but then it would be displayed on every page but I don't wanna have it displayed on my landingpage (index.js)

**Comment by NikRoe** [comment]
Ah, I see. Then I misunderstood what you intended to do.

**Pull request merged**